doc/sphinx/user/extending/plugin-types/postprocessors.md

```diff
@@ -123,56 +132,12 @@ Primarily, this difficulty results from two facts:
     certainly good examples to start from in trying to understand how to do
     this.
 
-Given these comments, the interface a postprocessor class has to implement is
-rather basic:
-
-```{code-block} c++
-template <int dim>
-    class aspect::Postprocess::Interface
-    {
-      public:
-        virtual
-        std::pair<std::string,std::string>
-        execute (TableHandler &statistics) = 0;
-
-        virtual
-        void
-        save (std::map<std::string, std::string> &status_strings) const;
-
-        virtual
-        void
-        load (const std::map<std::string, std::string> &status_strings);
-
-        static
-        void
-        declare_parameters (ParameterHandler &prm);
-
-        virtual
-        void
-        parse_parameters (ParameterHandler &prm);
-    };
-```
-
-The purpose of these functions is described in detail in the documentation of
-the `aspect::Postprocess::Interface` class. While the first one is
-responsible for evaluating the solution at the end of a time step, the
-`save/load` functions are used in checkpointing the program and restarting it
-at a previously saved point during the simulation. The first of these
-functions therefore, needs to store the status of the object as a string under
-a unique key in the database described by the argument, while the latter
-function restores the same state as before by looking up the status string
-under the same key. The default implementation of these functions is to do
-nothing; postprocessors that do have non-static member variables that contain
-a state need to overload these functions.
-
-There are numerous postprocessors already implemented. If you want to
-implement a new one, it would be helpful to look at the existing ones to see
-how they implement their functionality.
 
 ## Postprocessors and checkpoint/restart.
 
-Postprocessors have `save()` and `load()` functions that are used to write the
-data a postprocessor has into a checkpoint file, and to load it again upon
+Postprocessors have `save()` and `load()` functions (inherited from
+`aspect::Plugins::InterfaceBase` by way of `aspect::Postprocess::Interface`) that are used to write the
```

Comment:
It doesn't matter much, but the intent was to say that postprocessor classed get these functions not from the `Interface` class, but instead from the `InterfaceBase` class.

Reply:
Hmm, I feel like I do not understand what you mean. `InterfaceBase` does not declare or implement the `save()` and `load()` functions. They are specifically only declared and implemented in `Postprocess::Interface`, none of the other plugin systems have them.

Reply:
Oh, you're right. I totally misremembered. I must have assumed that all of the other plugins have these functions as well because they may want to save information too, but you're right that they don't -- apparently nobody ever had a need for saving information from the other plugins...